kin-devplatform-sdk/src/main/java/kin/devplatform/accountmanager/AccountManagerImpl.java

```diff
@@ -237,7 +245,59 @@ public void onError(Exception e) {
 	public void switchAccount(final int accountIndex, @NonNull final KinCallback<Boolean> callback,
 		@NonNull final IMigrationManagerCallbacks migrationManagerCallbacks) {
 		Logger.log(new Log().withTag(TAG).put("switchAccount", "start"));
-		startMigration(accountIndex, callback, migrationManagerCallbacks);
+		RestorableWalletApi restorableWalletApi = new RestorableWalletApi();
+		try {
+			handleRestorable(accountIndex, callback, migrationManagerCallbacks, restorableWalletApi);
+		} catch (ApiException e) {
+			Logger.log(new Log().priority(Log.ERROR).withTag(TAG).text("getIsRestorableWallet: error is " + e));
+			callback.onFailure(ErrorUtil.createWWalletWasNotCreatedInThisAppException());
```

Comment:
do not throe "wallet was not created in this app" here

Reply:
Same comment as above(regarding the progress dialog)